cs221bot.py

```diff
@@ -164,7 +169,14 @@ async def on_message(message):
     bot.loadJSON = loadJSON
     bot.writeJSON = writeJSON
 
-    for extension in filter(lambda f: isfile(join("cogs", f)), os.listdir("cogs")):
+    # The real Canvas API key (i.e. the one stored in the .env file) is required for Canvas
+    # courses without public access. However, the CPSC 221 course is accessible to the public.
+    # Since we only want to notify users about CPSC 221 assignments that are *publicly* available,
+    # we use the empty placeholder "" instead of the real Canvas API key when running in production.
+    # The -t flag is used in testing to indicate that we want to use the real Canvas API key.
+    bot.canvas_api_key = os.getenv("CANVAS_API_KEY") if args.testing_mode else ""
+
+    for extension in filter(lambda f: isfile(join("cogs", f)) and f != "__init__.py", os.listdir("cogs")):
```

Comment:
and same here as above, shouldn't be necessary if we can put `__init__.py` somewhere else